.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -16,7 +16,13 @@ labels: bug
 * then second step
 * I was expecting X but instead it shows Y
 
-Did you try to reproduce this issue at http://cdn.clappr.io/
+**Is this only happenning in your streaming?**
```

Comment:
Does it happen only in your streaming?

Reply:
@leandromoreira Thank you for the syntax improvements hints.